classes/class-wpcom-liveblog-rest-api.php

```diff
@@ -365,9 +433,10 @@ public static function get_single_entry( WP_REST_Request $request ) {
 	public static function format_preview_entry( WP_REST_Request $request ) {
 
 		// Get required parameters from the request
-		$post_id       = $request->get_param( 'post_id' );
-		$entry_content = $request->get_param( 'entry_content' );
-
+		$post_id 		= $request->get_param( 'post_id' );
+		$json 	 		= $request->get_json_params();
+		$entry_content  = self::get_json_param( 'entry_content', $json );
```

Comment:
Also here - how does `$request->get_json_params()` end up in `self::get_json_param()`?